src/modules/shortcut_guide/shortcut_guide.cpp

```diff
@@ -31,8 +33,8 @@ bool OverlayWindow::get_config(wchar_t* buffer, int* buffer_size)
     HINSTANCE hinstance = reinterpret_cast<HINSTANCE>(&__ImageBase);
 
     PowerToysSettings::Settings settings(hinstance, get_name());
-    settings.set_description(L"Shows a help overlay with Windows shortcuts when the Windows key is pressed.");
-    settings.set_overview_link(L"https://github.com/microsoft/PowerToys/blob/master/src/modules/shortcut_guide/README.md");
+    settings.set_description(GET_RESOURCE_STRING(IDS_SETTINGS_DESCRIPTION));
+    settings.set_overview_link(GET_RESOURCE_STRING(IDS_README_LINK));

```

Comment:
Is the README link also to be made into a resource? It wouldn't be localized so I'm not sure if it should be

Reply:
I also think it should be better to not localize the link and then have a Readme that has links for the localized version, so we can add more locales online without having to update the application.

Reply:
Reverted the change so that the link is not localized.